File: fixtures/matlab_output/multiple_d_post.m
url = 'https://cmdb.litop.local/webservices/rest.php';
body = 'version=1.2&auth_user=fdgxf&auth_pwd=oxfdscds&json_data={ "operation": "core/get", "class": "Software", "key": "key" }';
response = webwrite(url, body);

File: fixtures/elixir_output/multiple_d_post.ex
request = %HTTPoison.Request{
  method: :post,
  url: "https://cmdb.litop.local/webservices/rest.php",
  options: [],
  headers: [],
  params: [],
  body: [
    {~s|version|, ~s|1.2|},
    {~s|auth_user|, ~s|fdgxf|},
    {~s|auth_pwd|, ~s|oxfdscds|},
    {~s|json_data|, ~s|{ "operation": "core/get", "class": "Software", "key": "key" }|}
  ]
}

response = HTTPoison.request(request)
